test/system/500-networking.bats

```diff
@@ -798,5 +798,23 @@ EOF
     XDG_RUNTIME_DIR=$tmp_run run_podman run --network bridge --rm $IMAGE ip a
     assert "$output" =~ "eth0"
 }
+@test "podman inspect list networks " {
+    for network in "host" "none"; do
+        run_podman create --network=$network $IMAGE
+	cid=${output}
+	run_podman inspect --format '{{ .NetworkSettings.Networks }}' $cid
+	is "$output" "map\[$network:.*" "NeworkSettincs should container one network named $network"
+	run_podman rm $cid
+    done
+    run_podman create $IMAGE
+    cid=${output}
+    run_podman inspect --format '{{ .NetworkSettings.Networks }}' $cid
+    if is_rootless; then
+	is "$output" "map\[slirp4netns:.*" "NeworkSettings should container one network named slirp4netns"
+    else
+	is "$output" "map\[podman:.*" "NeworkSettings should container one network named podman"
+    fi
+    run_podman rm $cid
+}
```

Comment:
indentation looks wrong

Reply:
Not sure what you mean?

Reply:
look at the change in github, the indentation is off

Reply:
I don't see it.

Reply:
![image](https://user-images.githubusercontent.com/45212748/219038166-bf2fcafb-5996-4e30-82d4-a9bf8577fb0f.png)

This does not look right, I assume mixed tabs vs spaces. Ideally we would use a common formatter for these files they are always a mess to work with IMO cc @edsantiago